src/xhr.js

```diff
@@ -78,11 +135,18 @@ class Xhr {
 export function fetchPolyfill(input, opt_init) {
   assert(typeof input == 'string', 'Only URL supported: %s', input);
   const init = opt_init || {};
+
+  init.method = getMethod_(init);
```

Comment:
`getMethod_` does bunch of stuff assuming `init` can be undefined, but it's never the case, looks like?